packages/website/public/diagram-zoom.js

```
@@ -0,0 +1,63 @@
+// Click-to-enlarge for Mermaid diagrams (astro-mermaid renders into `.mermaid`).
+// Dependency-free: clones the rendered SVG into a native <dialog>, closes on
+// the button, a backdrop click, or Escape. Uses event delegation on `document`
+// so it survives client-side navigation and async diagram rendering.
+(function () {
+  if (window.__diagramZoom) return;
```

ESLint

[error] 'window' is not defined.

(no-undef)